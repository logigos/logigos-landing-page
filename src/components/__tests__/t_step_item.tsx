import React from 'react'
import renderer from 'react-test-renderer'
import StepItem from '../common/StepFeature'
import IconAnnouce from '../../images/step-section/icon/annouce.svg'

describe('StepItem', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(
                <StepItem
                    icon={<IconAnnouce />}
                    name={'test_name'}
                    detail={'test_detail'}
                />
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
