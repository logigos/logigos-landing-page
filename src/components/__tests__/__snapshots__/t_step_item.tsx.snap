// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepItem renders correctly 1`] = `
<div
  className="sc-AxirZ sc-AxiKw sc-Axmtr caIeeO"
>
  <test-file-stub />
  <p
    className="sc-fzozJi gLGUah"
  >
    test_name
  </p>
  <p
    className="sc-fzoLsD kGRssa"
  >
    test_detail
  </p>
</div>
`;
